### DISCUSSION TITLE
what is Build-a-thon?

### DISCUSSION THREAD

**Discussion by CrashingGuru** [vrra/FGAN-Build-a-thon-2022#7] [Q&A (General)] [answered]
what is Build-a-thon?

**Comment by CrashingGuru** [comment, marked as answer]
The build-a-thon is a problem statement in ITU AI/ML in 5G Challenge hosted by [ITU-T Focus Group Autonomous Networks](https://www.itu.int/en/ITU-T/focusgroups/an/Pages/default.aspx). 
It aims to demonstrate and validate important use cases for autonomous networks, creating proof of concept implementations and tools in the process. As an open platform, FG AN is well-poised to enable access to experts, students and industry, collaborate with other external events such as plugfests and hackathons and to set the stage for collaboration in open-source projects, other PoC (proof of concept) and standardization work.

In essence, a build-a-thon is a common, open platform for like-minded people to come together (remotely) and build something to prove a point. In the case of FG AN we define it as below:
1. Build-a-thon is a PoC development activity, to build upon a key concept in FG AN, especially intended to prove the concept practically with code, test setup and demo setup.
2. Build-a-thon is not intended to create a product, nor would the code created as part of Build-a-thon be considered as product quality software.
3. Build-a-thon would create well-documented artefacts and opensource code.
<img width="845" alt="image" src="https://user-images.githubusercontent.com/20972784/164420854-0548ba60-418e-4102-96be-d10829ade777.png">

For more details see : https://extranet.itu.int/sites/itu-t/focusgroups/an/input/FGAN-I-239.docx

**Reply by ashokkamaraj** [reply]
For this FGAN-Build-a-thon, if we have selected a use-case how to get data, do you know ?

For ex: If I have selected following use cases, 
•	Open, integrated, log analysis - How to get logs from various vendor 5G NE logs
•	Intelligent energy saving for data centres - How to get Energy Saving input configurations to predict when to switch on / off, or which cell kind of.
